packages/pn-commons/src/utils/onetrust.utility.ts

```diff
@@ -0,0 +1,7 @@
+export const ONE_TRUST_INSTANCE_ID = '77f17844-04c3-4969-a11d-462ee77acbe1';
+export const ONE_TRUST_BASEPATH = `https://privacyportalde-cdn.onetrust.com/${ONE_TRUST_INSTANCE_ID}/privacy-notices`;
```

Comment:
Maybe we can use only one variable ONE_TRUST_CDN. ONE_TRUST_INSTANCE_ID is not used outside this file